build.rs

```diff
@@ -11,6 +11,10 @@ fn main() {
         autocfg::emit("has_i128");
     }
 
+    if env::var_os("CARGO_FEATURE_COPYSIGN").is_some() || ac.probe_path("f32::copysign") {
```

Comment:
`f32::copysign` is not a valid path, or at least it's not one that works with `use PATH` as attempted by `probe_path`. You should be able to `probe_expression` though.

(`cargo check -vv` is helpful to see build script output -- this path is failing even on current stable.)